Server/htdocs/AppController/commands_RSM/itemsManager/classLbxUsers_createUser.php

```diff
@@ -3,13 +3,27 @@
 require_once "../utilities/RSdatabase.php";
 require_once "../utilities/RSMitemsManagement.php";
 require_once "../utilities/RSMbadgesManagement.php";
+require_once "../utilities/RSMbadgesManagement.php";
 
 $clientID =               $GLOBALS['RS_POST']['clientID'] ;
 $login    = base64_decode($GLOBALS['RS_POST']['login'   ]);
 $password =               $GLOBALS['RS_POST']['password'] ;
 $personID =               $GLOBALS['RS_POST']['personID'] ;
+$badge =                  $GLOBALS['RS_POST']['badge'   ] ;
```

Comment:
The badge should be an optional parameter. If it is not sent by RSM the user should be created anyway with a random badge.